### PR TITLE
Only show dev registry connection status in local dev

### LINKED ISSUE #7009: FEEDBACK: red or green dot on DO services
In PR https://github.com/cloudflare/workers-sdk/pull/6893, a red or green dot was added beside each service when deploying a DO with Wrangler.

I'm not too familiar with Wrangler internals, but it looks like the red vs. green is decided based on whether a service is in the registry.

Per some feedback on [Discord](https://canary.discord.com/channels/595317990191398933/779390076219686943/1296367336462225469), this isn't really clear to users, and in fact the random red dot can be kind of concerning given that there is no explanation on what it means or why your services are suddenly red.

I believe it was added in PR https://github.com/cloudflare/workers-sdk/pull/6893:
https://github.com/cloudflare/workers-sdk/blob/ae4bfead3ad9911823ec154476cb1c49ea3a02bf/packages/wrangler/src/config/index.ts#L268

### PR DESCRIPTION
**Pull request by penalosa**
Fixes #7009

---

- Tests
  - [ ] TODO (before merge)
  - [x] Tests included
  - [ ] Tests not necessary because:
- E2E Tests CI Job required? (Use "e2e" label or ask maintainer to run separately)
  - [ ] I don't know
  - [x] Required
  - [ ] Not required because:
- Public documentation
  - [ ] TODO (before merge)
  - [ ] Cloudflare docs PR(s): 
  - [x] Documentation not necessary because: small output change

### CODE REVIEW

File: packages/wrangler/src/config/index.ts [hunk 1]
```diff
@@ -264,16 +264,21 @@ export function printBindings(
...
+									(d) => d.className === class_name
+								)
+							) {
+								value += ` (defined in 🟢 ${script_name})`;
+							} else {
+								value += ` (defined in 🔴 ${script_name})`;
+							}
 						} else {
-							value += ` (defined in 🔴 ${script_name})`;
+							value += ` (defined in ${script_name})`;
```
Comment (CarmenPopoviciu) [inline packages/wrangler/src/config/index.ts:281]: what's the logic based on which we apply these red/green bullets? Cause it seems we apply them in some places but no in others?

Comment (penalosa) [reply packages/wrangler/src/config/index.ts:281]: Before this PR we applied them to all DOs & service bindings based on whether they were in the dev registry. After this PR we apply to all DOs & service bindings based on whether they were in the dev registry, but hide them if not in local dev

File: packages/wrangler/src/config/index.ts [hunk 2]
```diff
@@ -264,16 +264,21 @@ export function printBindings(
...
+
+							if (
+								registryDefinition &&
+								registryDefinition.durableObjects.some(
+									(d) => d.className === class_name
+								)
+							) {
+								value += ` (defined in 🟢 ${script_name})`;
+							} else {
+								value += ` (defined in 🔴 ${script_name})`;
```
Comment (CarmenPopoviciu) [inline packages/wrangler/src/config/index.ts:278]: now that I've read the actual issue 😅 , I'm seeing that our users' confusion is around using the red bullets. Do we still want to use them in this place in that case?

Comment (penalosa) [reply packages/wrangler/src/config/index.ts:278]: I think this is a good first step—it removes them from the dploy flow, which is where I think most confusion was coming from

Comment (CarmenPopoviciu) [reply packages/wrangler/src/config/index.ts:278]: kk

**Pull request merged**